packages/editor/src/services/EditorServices.ts

```diff
@@ -86,5 +89,17 @@ export const EditorState = defineState({
     if (!rootEntity) return false
     return !!getState(GLTFModifiedState)[getComponent(rootEntity, SourceComponent)]
   },
-  extension: syncStateWithLocalStorage(['expandedNodes'])
+  extension: syncStateWithLocalStorage(['expandedNodes']),
+  reactor: () => {
+    const linkState = useMutableState(LinkState)
```

Comment:
ideally this should not be the responsibility if the EditorState, this should be handled in the link opening logic in client-core

Reply:
The EngineHooks.tsx file in client-core that handles the scene link logic isn't loaded through the studio flow, I can make it a part of the engine loading flow for the studio though